backend/src/main/resources/db/migration/postgresql/V10__Link_Executant_to_Facility.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+ALTER TABLE executant
+    ADD COLUMN facility_id bigint references facility (id);
+
+UPDATE executant SET facility_id=1 WHERE facility_id IS NULL;
```

Comment:
alter table executant add column facility_id bigint references facility(id) not null default 1;
alter table executant alter column facility_id drop default;

Reply:
Wanted to explicitly state the assignment of facilities, but my way doesn't account of entries being inserted between `UPDATE` AND `SET NOT NULL` statements.

Will use your proposed approach